wp-content/themes/citylimits/tribe-events/widgets/list-widget.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Events List Widget Template
+ * This is the template for the output of the events list widget.
+ * All the items are turned on and off through the widget admin.
+ * There is currently no default styling, which is needed.
+ *
+ * This view contains the filters required to create an effective events list widget view.
+ *
+ * You can recreate an ENTIRELY new events list widget view by doing a template override,
+ * and placing a list-widget.php file in a tribe-events/widgets/ directory
+ * within your theme directory, which will override the /views/widgets/list-widget.php.
+ *
+ * You can use any or all filters included in this file or create your own filters in
+ * your functions.php. In order to modify or extend a single filter, please see our
+ * readme on templates hooks and filters (TO-DO)
+ *
+ * @version 4.4
+ * @return string
+ *
+ * @package TribeEventsCalendar
+ *
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+$events_label_plural = tribe_get_event_label_plural();
+$events_label_plural_lowercase = tribe_get_event_label_plural_lowercase();
+
+$posts = tribe_get_list_widget_events();
```

Comment:
Regarding my comment on the `tribe_custom_list_widget_events` filter function, could we perhaps use that filter's query args here, replacing this function with a different query, or passing those query args directly to this function instead of relying on a filter?